tests/unit/lms/services/lti_outcomes_test.py

```diff
@@ -214,63 +166,50 @@ def test_it_gracefully_handles_RequestException(
         with pytest.raises(LTIOutcomesAPIError):
             lti_outcomes_svc.read_result(lti_outcomes_params)
 
-    @pytest.fixture
-    def request_xml(self):
-        """Return parsed XML of last request."""
-
-        def xml():
-            request = httpretty.last_request()
-            return ElementTree.fromstring(request.body)
-
-        return xml
-
-    @pytest.fixture
-    def record_result_request_fields(self, request_xml):
```

Comment:
This function is very bad for understanding the tests. Each of the things here pretty much only relate to one test but are grouped together here.

Reply:
Yeah, that sounds terrible, and now how fixtures should be used